docs/user_guide/assessment.rst

```diff
@@ -258,6 +258,124 @@ across groups and also the difference and ratio between the maximum and minimum:
     >>> print("ratio in recall = ", grouped_metric.ratio(method='between_groups'))    
     ratio in recall =  0.0
 
+
+Fairness metrics
+----------------
+
+Parity constraints
+^^^^^^^^^^^^^^^^^^
+In the sections below, we review the most common fairness metrics, as well
+as their underlying assumptions and suggestions for use. Each metric requires
+that some aspects of the predictor behavior be comparable across groups. In 
+the mathematical definitions below, let :math:`X` denote a feature vector 
+used for predictions, :math:`A` be a single sensitive feature (such as age 
+or race), and :math:`Y` be the true label. Parity metrics are phrased in 
+terms of expectations with respect to the distribution over :math:`(X,A,Y)`.
+
+Demographic parity
+~~~~~~~~~~~~~~~~~~
+Demographic parity is also known as *statistical parity*. A classifier
+:math:`h` satisfies demographic parity under a distribution over
+:math:`(X, A, Y)` if its prediction :math:`h(X)` is statistically
+independent of the sensitive feature :math:`A`. This is equivalent to
+:math:`\E[h(X) \given A=a] = \E[h(X)] \quad \forall a`. :footcite:`agarwal2018reductions`
+In the case of regression, a predictor :math:`f` satisfies demographic parity
```

Comment:
Note that in the Fairlearn API, `demographic_parity_difference` is only defined for the classification scenario. I do think it is worthwhile to mention regression, but it might be better off as a separate paragraph within the `demographic parity` section, where we highlight that you can compute `mean_prediction()` across groups as one way to measure allocation harms in the regression context. Again, adding examples is important for users to understand why this might matter.

Reply:
I added another paragraph underneath the classification scenario.